airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -476,6 +476,7 @@ protected List<ConfiguredAirbyteStream> identifyStreamsToSnapshot(final Configur
     final Set<AirbyteStreamNameNamespacePair> newlyAddedStreams = new HashSet<>(Sets.difference(allStreams, alreadySyncedStreams));
 
     return catalog.getStreams().stream()
+        .filter(c -> c.getSyncMode() == SyncMode.INCREMENTAL)
```

Comment:
Is this a bug in the current implementation of CDC initial stream snapshotting? Are we not filtering based on whether the stream is in incremental mode?